pkg/types/types.go

```diff
@@ -344,6 +345,15 @@ type CA struct {
 	Password   string `yaml:"password,omitempty"`
 }
 
+type Thanos struct {
+	Disabled               bool   `yaml:"disabled,omitempty"`
+	Version                string `yaml:"version,omitempty"`
+	Mode 	               string `yaml:"mode,omitempty"`
+	ThanosSidecarEndpoint  string `yaml:"thanosSidecarEndpoint,omitempty"`
+	ThanosSidecarPort      string `yaml:"thanosSidecarPort,omitempty"`
+	S3                    *S3     `yaml:"s3,omitempty"`
```

Comment:
I think this should be a bucket name only and then we piggy-back off platform.S3 for credentials

Reply:
There can't be multiple buckets for Thanos Querier. So we can't use local Minio for storing metrics for every client cluster. Or platform.S3 is the same for all clusters?

Reply:
Yes, platform.S3 should point to the same object store (minio / S3) for all clusters.